data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
+					decline
+			label yes
+			`You raise your hand and one of the soldiers notices you.`
+			`"You? Why not. Try not to hurt yourself."`
+			`You got the job.`
+				accept
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis marked
+			confusion 30
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 1
+	on complete
+		payment 20000
+		dialog `A Navy soldier approaches you. "Well done!", and with a firm hand shake she hands you <payment>.`
+
+mission "Renegade Drones 2"
+	name "Neutralize broken Drones"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drones."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 10
+		"Renegade Drones 1: done" > 3
+		"combat rating" > 600
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`Walking through the space port, you are approached by a Navy officer. "Captain <last>?" she asks. You nod. "I've heard you're quite good at shooting things from the sky. A group of drones we've tested experienced a malfunction. Now they rampage through the system attacking everything getting in their way. Unfortunately, we lack personnel at the moment. Would you be willing to help us out and shoot down the malfunctioning drones?"`
+			choice
+				`	"Sure, I always welcome targeting practice."`
+					accept
+				`	"No, sorry, I'm busy right now."`
+					decline
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis
+			confusion 30
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 10
+	npc evade
+		government "Renegade"
+		personality waiting heroic vindicitive target
+			confusion 30
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 30
+	npc
+		personality heroic staying uninterested
+		government "Republic"
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 20
+	on complete
+		payment 200000
+		dialog "You got the drones under control and collect your payment of <payment>."
+
+mission "Renegade Drones 3"
+	name "Neutralize AI Drones"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drones."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 5
+		"Renegade Drones 2: done" > 3
+		"combat rating" > 1200
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`You abruptly turn around when a rough but very distinct voice calls your name: "Captain <last>! Been looking for you." You stand in front a senior Navi Captain. "One says, you have quite some experience with drones going berzerk. Apparently, those guys from the lab meant it a little bit to well with the artificial intelligence in our newest drone model. It seems it started wondering, why it should work for us. Would you be willing to, let's say, correct their mistake?"`
```

Comment:
@tehhowch: Not sure which part you mean. Maybe "It seems it started wondering, why it should work for us." ? English isn't my first language and I can't find a rule where reported speech is set in quotation marks. Could you elaborate?
@Bladewood: Not sure about "Hey! Listen! /joke" ... That's an actual joke, not a suggestion for improvement? Slightly confused about the unknown culture reference ...